Dockerfile.test

```diff
@@ -0,0 +1,58 @@
+# Dockerfile for testing swarm
```

Comment:
I think this Dockerfile could be located in `test/integration` (and simply named `Dockerfile`) and take care of only running integration tests (we are still using travis for unit)